res/values-ko/strings.xml

```diff
@@ -1311,17 +1311,17 @@ Nebulous 규칙\n
     <string name="The_winner_will_receive_x_plasma_">승자는 %s개의 플라즈마를 획득합니다!</string>
     <string name="admin_commands">"
 명령어:
-!kick [0–31]
-!reset
-!welcome {message}
-!difficulty [0-3] or [1-4] for Campaign
-!hidden
-!guests
-!slowchat
-!pausechat
-!mute {id}
-!admin {id}
-!allow {id}
+!kick [0–31] [0-31] 추방 명령어
```

Comment:
Why repeated the "[0-31]"?

Reply:
Tip from me also, to explain **every** command. !allow reset and !winner has been abandoned.